GNUmakefile

```diff
@@ -69,7 +69,12 @@ endif
 
 tool_targets += tools/hookgen
 
-HOOK_DEFINITION_FILES = $(abspath ./gc/base/omrmmprivate.hdf ./gc/include/omrmm.hdf ./fvtest/algotest/hooksample.hdf)
+# convert Cygwin path to Windows path with regular slashes
+define convertUnixToWinPath
+$(if $(findstring CYGWIN,$(shell uname -s)),$(foreach f,$1,$(shell cygpath -m $f)),$1)
+endef
```

Comment:
I suggest the definition of this macro should be conditional rather than invoking `uname` for each use.
Also, cygpath can handle all paths in a single invocation so `foreach` is not needed here.

Reply:
@keithc-ca Fixed as suggested. Also renamed it to convertPath.